docs/src/main/asciidoc/getting-started-dev-services.adoc

```diff
@@ -0,0 +1,326 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+
+[id="getting-started-dev-services-tutorial"]
+= Your second Quarkus application
+include::_attributes.adoc[]
+:categories: getting-started, data, core
+
+
+This tutorial shows you how to create an application which writes to and reads from a database.
+You will use Dev Services, so you will not actually download, configure, or even start the database yourself.
+You will use Panache to make reading and writing data easier.
+
+
+== Prerequisites
+
+:prerequisites-time: 30 minutes
+:prerequisites-docker:
+:prerequisites-no-graalvm:
+include::{includes}/prerequisites.adoc[]
+
+This tutorial builds on what you learned writing xref:{doc-guides}/getting-started.adoc[your first Quarkus application].
+You will not need the code from that application, but make sure you understand the concepts.
+
+== Solution
+
+We recommend that you follow the instructions from <<Bootstrapping the project>> onwards to create the application step by step.
+
+However, you can go right to the completed example.
+
+Download an {quickstarts-archive-url}[archive] or clone the git repository:
+
+[source,bash,subs=attributes+]
+----
+git clone {quickstarts-clone-url}
+----
+
+The solution is located in the `getting-started-dev-services` {quickstarts-tree-url}/getting-started-dev-services[directory].
+
+:sectnums:
+:sectnumlevels: 3
+== Outline steps
+
+- Bootstrap the application
+- Update the application to read user input
+- Create a Panache Entity
+- Read and write the entity
+- Configure an external database using a profile
+
+== Setting up an interactive application
+
+=== Bootstrapping the project
+
+The easiest way to create a new Quarkus project is to open a terminal and run the following command:
+
+For Linux & MacOS users
+
+:create-app-artifact-id: getting-started-dev-services
+:create-app-extensions: resteasy-reactive
+:create-app-code:
+include::{includes}/devtools/create-app.adoc[]
+
+For Windows users:
+
+- If using cmd , (don't use backward slash `\` and put everything on the same line)
+- If using Powershell , wrap `-D` parameters in double quotes e.g. `"-DprojectArtifactId=getting-started"`
+
+For an explanation of what's in the generated application, see the xref:getting-started.adoc[First application guide].
+
+=== Running the application
+
+Launch the application in dev mode
+----
+mvn quarkus:dev
+----
+
+Once the application is up, visit http://localhost:8080/hello. It should show a "Hello from RestEasy Reactive" message.
+
+=== Accepting user input
+
+Let's make the application a bit more interactive.
+Open the project in your IDE and navigate to `src/main/java/org/acme/GreetingResource.java'
+Add a query param in the `hello` method.
+
+----
+public String hello(@QueryParam("name") String name) {
```

Comment:
Use the RESTEasy Reactive one to avoid repeating the `name`.

Also, mention the fully qualified class name.

Reply:
Ooh, I'm not sure I knew about that one. I've added to my 'list of boilerplate reductions' slides.